src/proxy-ui-api/frontend/tests/e2e/specs/ss-settings-timestamping.js

```diff
@@ -27,6 +27,7 @@
 let mainPage, settingsTab, systemParametersTab;
 
 module.exports = {
+  '@disabled': true, // assumes timestamping is not set yet, which is not true in aws emvironment
```

Comment:
Since timestamping is already fixed, this probably needs to be removed before merging.

Reply:
I think I will merge develop (including the timestamping fixes) to this and do some kind of endurance test run first. I guess timestamping tests pass but haven't been through endurance tests yet.

Reply:
Reliable results from endurance tests, 100% success for https://jenkins.niis.org/view/api-based-ui/job/rest-ui-e2e-tests/ jobs 749 - 761